islandora_document.install

```diff
@@ -22,3 +22,27 @@ function islandora_document_uninstall() {
   module_load_include('inc', 'islandora', 'includes/solution_packs');
   islandora_install_solution_pack('islandora_document', 'uninstall');
 }
+
+/**
+ * Update Islandora document viewers selection.
+ *
+ * The viewer to display the preview of a document object is currently
+ * determined by the viewer selected in the Islandora PDF solution pack
+ * configuration, rather than the Islandora document solution pack
+ * configuration.
+ *
+ * This update sets the Islandora document solution pack viewer to the same
+ * value as the Islandora PDF solution pack viewer so that the fix in
+ * theme/theme.inc to use the viewer selected in the Islandora document admin
```

Comment:
The slash here is being stripped on render.  Perhaps we don't need to mention the file?